src/App.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+import Stats from "./components/Stats";
+import Trends from "./components/Trends";
+import ExternalLinks from "./components/ExternalLinks";
+import AdviceLine from "./components/AdviceLine";
+import Footer from "./components/Footer";
+import data from "./data/data";
+import "./App.scss";
+
+const Title = () => (
+  <>
+    <h1>Brunei COVID-19 Tracker</h1>
+
+    <div className="last-updated">
+      Last Updated:{" "}
+      {new Date(data.lastUpdated).toLocaleString("en-SG", {
```

Comment:
This should run at build instead of runtime.

Reply:
resolved as suggested [using this](https://stackoverflow.com/questions/53028778/how-to-show-build-datetime-on-my-react-web-app-using-create-react-app)